lib/template-generator.js

```diff
@@ -12,7 +12,7 @@ module.exports = function templateGenerator (opts, cb) {
   var dateFormat = opts.dateFormat
   var templateFile = opts.templateFile || path.join(__dirname, 'template.js')
   var migrationsDirectory = opts.migrationsDirectory || 'migrations'
-  var extension = opts.extension || '.js'
+  var extension = opts.extension || opts.extention || '.js'
```

Comment:
Can you correct the option at the `bin` script level?  Inside that if statement would be best.  Then remove this one.